redwood-widget-composeui/build.gradle

```diff
@@ -0,0 +1,30 @@
+import app.cash.redwood.buildsupport.KmpTargets
+
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'com.android.library'
+
+redwoodBuild {
+  composeCompiler()
+  publishing()
+}
+
+kotlin {
+  androidTarget {
+    publishLibraryVariants('release')
+  }
+
+  KmpTargets.addAllTargets(project, true /* skipJs */)
+
+  sourceSets {
+    commonMain {
+      dependencies {
+        implementation libs.jetbrains.compose.foundation
+        implementation projects.redwoodRuntime
```

Comment:
```suggestion
        api projects.redwoodRuntime
```
For `Margin`

Reply:
I've moved the insets over to `redwood-composeui`. `redwood-composeui` has an `api` dependency on `redwood-compose`, which has an `api` dependency on `redwood-runtime`, so we no longer need to explicitly specify a dependency on `redwood-runtime`.